src/main/java/teammates/ui/webapi/action/DeleteInstructorInCourseAction.java

```diff
@@ -55,20 +56,31 @@ public ActionResult execute() {
     private boolean hasAlternativeInstructor(String courseId, String instructorToDeleteEmail) {
 
         List<InstructorAttributes> instructors = logic.getInstructorsForCourse(courseId);
+        boolean isAlternativeInstructor = false;
+        boolean isAlternativeInstructorDisplayed = false;
```

Comment:
The variable names are not very accurate. Something like `isAlternativeInstructorWithModifyInstructorPermissionPresent` (might be too long) and `isAlternativeVisibleInstructorPresent` are more representative of what they're supposed to represent.